docs/rcweb/requirements.txt

```
@@ -1,5 +1,5 @@
-reflex==0.5.9
-reflex-chakra==0.1.1a1
+reflex==0.6.2
```

we might as well specify an upper bound with the `reflex` and `reflex-chakra` versions
```suggestion
reflex>=0.6.2
```